components/Slider/tests/snapshot/MDCSliderSnapshotTests.m

```diff
@@ -55,7 +82,7 @@ - (void)setUp {
 
   // Uncomment below to recreate all the goldens (or add the following line to the specific
   // test you wish to recreate the golden for).
-  // self.recordMode = YES;
+  //  self.recordMode = YES;
```

Comment:
nit: revert

Reply:
The new formatting matches the typical formatting for these files.  If you're adamant about splitting this change to a separate PR, I can do it, but I don't feel it's worth rerunning presubmits.

Reply:
Done, latency was too high.